src/component-library/TokenFieldWithBalance/TokenFieldWithBalance.tsx

```diff
@@ -0,0 +1,39 @@
+import * as React from 'react';
+
+import { TokenFieldWithBalanceWrapper } from './TokenFieldWithBalance.style';
+import { TokenField, TokenFieldProps } from 'component-library/TokenField';
+import { Balance, BalanceProps } from 'component-library/Balance';
+
+// ray test touch <
+// TODO:
+// - `label` and `currencySymbol` are duplicated.
+// - `approxUSD` and `usdValue` are duplicated.
+// - `currencyValue` and `value` are duplicated.
+interface TokenFieldWithBalanceProps extends TokenFieldProps, BalanceProps {}
+// ray test touch >
+
+// ray test touch <
+// TODO:
+// - `CurrencySymbols` type should be used for `label` of `TokenField`.
+// ray test touch >
+
+const TokenFieldWithBalance = React.forwardRef<HTMLInputElement, TokenFieldWithBalanceProps>(
+  ({ currencySymbol, currencyValue, usdValue, approxUSD, label, value, ...rest }, ref): JSX.Element => {
+    // ray test touch <
+    // TODO:
+    // - `approxUSD` -> `usdValue` | `valueInUSD`
```

Comment:
Hi @peterslany 
There are some differences in using terms so I'd like them to be consistent.
- I'm using `approxUSD`.
- You're using `usdValue`.
I think yours sounds better. What do you think of `valueInUSD`? A little verbose?

Reply:
I made them consistent across `TokenBalance`, `TokenField`, and `TokenFieldWithBalance`.